modules/KIWILinuxRC.sh

```diff
@@ -10525,6 +10525,15 @@ function pxeRaidAssemble {
         fi
         IFS=$IFS_ORIG
         mdadm --assemble --run /dev/md$mdcount $devices
```

Comment:
seems my comment by mail got lost

for sle12 you are correct, but this is the v7 code base and that is also used for sle11 builds. Thus I don't want to break it